ToolBox/Program.cs

```diff
@@ -255,6 +255,14 @@ public static void Main(string[] args)
                             GetParameterOrExit(optionsObject, "source-dir"),
                             GetParameterOrExit(optionsObject, "destination-dir"));
                         break;
+                    case "sidc":
+                    case "smartinsiderconverter":
+                        SmartInsiderProgram.SmartInsiderConverter(
+                            DateTime.ParseExact(GetParameterOrExit(optionsObject, "date"), "yyyyMMdd", CultureInfo.InvariantCulture),
```

Comment:
Use `CultureInfo.InvariantCulture` -- or -- if #3442 is merged in time, use `StringExtensions.ParseDateTimeExactInvariant(string str, string format)`